ios/Plugin/Helpers/PluginGasStation.swift

```diff
@@ -31,8 +34,40 @@ struct PluginGasStation: Codable {
         self.isConnectedFuelingAvailable = poiStation.isConnectedFuelingAvailable
         self.lastUpdated = lastUpdated
         self.address = Address(from: attributes.address)
+        self.fuelPrices = poiStation.prices.compactMap { FuelPrice(from: $0, format: attributes.priceFormat) }
+        self.openingHours = parseOpeningHours(from: attributes.openingHours)
     }
 
+    private func parseOpeningHours(from poiOpeningHours: PCPOICommonOpeningHours?) -> [OpeningHour] {
+        guard let poiOpeningHours = poiOpeningHours,
+              let rules = poiOpeningHours.rules
+        else { return [] }
+
+        let openingHours: [OpeningHour] = [PCPOICommonOpeningHours.Rules.PCPOIDays.monday,
```

Comment:
Is this some sort of code style to have the first enum including the whole namespace or was the compiler somehow confused here?

Reply:
Otherwise the type cannot be inferred, at least that's how I do it in these cases but I forgot about it being `CaseIterable` -> adjusted it